src/Engines/QueryBuilderEngine.php

```diff
@@ -378,6 +378,34 @@ protected function compileQuerySearch($query, $column, $keyword, $relation = 'or
         $column = $this->castColumn($column);
         $sql    = $column . ' LIKE ?';
 
+        /**
+         * Patch for fix about ambiguous field
+         *
+         * Ambiguous field error will appear
+         * when query use join table and search with keyword.
+         */
+        // Remove backtick that appear from MYSQL query.
+        $column = str_replace('`', '', $column);
```

Comment:
Is this part necessary? Other DB uses `"` double quote to escape the columns. I think MSSQL uses brackets like `[column]`?